docs/topic-guides/web-apps/app-shell.md

```diff
@@ -19,6 +19,7 @@ The Starcounter app shell is a minimalist, configurable HTML document that impor
 *  the `<starcounter-include>` custom element
 *  Underwear.css, a CSS framework
 *  a placeholder for the Web App Manifest \(`link rel="manifest"`\).
+*  `starcounter.html` file that contains script, which automatically adds the `aria-current-"page"` attribute to links, which for example, you can use to highlight the current link in the menu, like the [Uniform.css](https://docs.starcounter.io/v/2.4/topic-guides/blendable-web-apps/starcounter-design-system#uniform.css-ui-patterns-in-a-separated-presentation-layer) `uni-layout-left-nav` class does.
```

Comment:
Perfect! Just two tiny things: "that contains a script" and the comma should be after "which": "to links which, for example, "

Reply:
Done!